Directory.Build.props

```diff
@@ -239,6 +239,20 @@
     <ArchiveExtension Condition="'$(TargetOsName)' == 'win'">.zip</ArchiveExtension>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- PackageReadmeFile specifies the package readme file name in the package. PackageReadmeFilePath points to the package readme file on disk. -->
+    <EnableDefaultPackageReadmeFile Condition="'$(EnableDefaultPackageReadmeFile)' == '' and '$(IsShipping)' != 'false'">true</EnableDefaultPackageReadmeFile>
+    <PackageReadmeFilePath Condition="'$(PackageReadmeFilePath)' == '' and '$(EnableDefaultPackageReadmeFile)' == 'true'">PACKAGE.md</PackageReadmeFilePath>
+    <PackageReadmeFile Condition="'$(PackageReadmeFile)' == '' and '$(EnableDefaultPackageReadmeFile)' == 'true' and Exists('$(PackageReadmeFilePath)' )">PACKAGE.md</PackageReadmeFile>
+    <!-- Uncomment this once all packages have ReadMes -->
+    <!-- <BeforePack>$(BeforePack);ValidatePackageReadmeExists</BeforePack> -->
```

Comment:
It doesn't look like the `ValidatePackageReadmeExists` target is defined in `aspnetcore`. The runtime repo defines it themselves [here](https://github.com/dotnet/runtime/blob/1231a9addd0fb3cfb806c2a89cb45a3018c48bb2/eng/packaging.targets#L336), should we be doing the same? Or does it come from somewhere else?

Reply:
Or is the plan to also wait to add that target until we're ready to uncomment the line?

Reply:
Oh, good catch, I'll just remove it in favor of the defensive conditions I already added